main.go

```diff
@@ -222,7 +223,17 @@ func configureLogger(clogger zap.Logger, config server.Config) (zap.Logger, zap.
 
 func dbConnect(multiLogger zap.Logger, dsns []string) *sql.DB {
 	// TODO config database pooling
-	db, err := sql.Open("postgres", "postgresql://"+dsns[0]+"/nakama?sslmode=disable")
+	rawurl := fmt.Sprintf("postgresql://%s?sslmode=disable", dsns[0])
+	url, err := url.Parse(rawurl)
+	if err != nil {
+		multiLogger.Fatal("Bad connection URL", zap.Error(err))
+	}
+
+	if len(url.Path) < 1 {
+		url.Path = "/nakama"
```

Comment:
What line do you refer to earlier in the migration? Have a look at - https://github.com/heroiclabs/nakama/pull/18/files#diff-f1bf5486c350dca6f6f470fa77823555R136